components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -67,12 +68,22 @@ export default function Preferences() {
         [toast, setUser, workspaceTimeout],
     );
 
+    const clearCachedStartWorkspaceOptions = useCallback(async () => {
+        if (!user) {
+            return;
+        }
+        AdditionalUserData.set(user, { workspaceCreationOptionsPerCloneURL: [] });
+        setUser(user);
+        await getGitpodService().server.updateLoggedInUser(user);
+        toast("Your cached start options were cleared.");
```

Comment:
**nitpick**: Minor rephrase to match the terms proposed below.

```suggestion
        toast("Your default workspace start options were cleared.");
```

Reply:
It's strange to have a "Reset Start Options" section and "New Workspaces" section when this control is related to "New Workspaces".

Also, we should be consistent when saying "default workspace start options", "remembered", "cached", "saved", etc. I suggest we call these Autostart options to match the URL parameter `autoStart=true`.


<img width="531" alt="image" src="https://user-images.githubusercontent.com/30455/235778455-85a59bf8-de72-4a60-9165-9e2286d5ad8d.png">